src/Entity/Type/RefundEnumType.php

```diff
@@ -64,4 +64,11 @@ protected function createType(string $value): RefundTypeInterface
     {
         return new RefundType($value);
     }
+
+    public function requiresSQLCommentHint(AbstractPlatform $platform)
```

Comment:
We should add a return type here `: bool` as well as put it before `createType` function to keep methods order